### DISCUSSION TITLE
See how to improve performance

### DISCUSSION THREAD

**Discussion by drcucaracha** [GodotECS/godex#269] [Ideas & Proposals]
Hello, I wanted to know if you could clarify a question, it is possible to change the rendering of godot for the Magnum ( https://github.com/mosra/magnum ) library that is data-oriented and now they are adapting it to use EnTT ( https://github.com/skypjack/entt ) as a complement or it is something impossible, thanks for your time to read this :)

**Comment by AndreaCatania** [comment]
This is definitely doable, there are some challenges, but doable. It would be interesting to demo something like that!
